src/Aks/Aks/ChangeLog.md

```diff
@@ -19,6 +19,8 @@
 -->
 ## Upcoming Release
 
+* [Breaking Change] Updated Azure RBAC Contributor access scope to target the Managed Cluster resource group instead of subscription-level in `New-AzAksCluster`.
```

Comment:
The change log in this PR should looks:
```suggestion
* Added notification that will Azure RBAC Contributor access scope to target the Managed Cluster resource group instead of subscription-level in `New-AzAksCluster` in next marjor release.
```